### PR TITLE
Home page fixes

### PR DESCRIPTION
**Pull request by Sharonsyra**
**What does this PR do?** 

- It changes the look of the banner and improves the carousel 

**Description of tasks**

- A text section was added on the banner
- The button was placed on the middle of the section
- Carousel look and feel has improved
- Changes to partners' slider
- Changes to footer
- Add blank target to anchor tags

screen shot 2017-09-18 at 3 54 05 pm
screen shot 2017-09-20 at 8 59 36 am
screen shot 2017-09-20 at 8 58 19 am

### CODE REVIEW

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html [hunk 1]
```diff
@@ -25,15 +25,28 @@ <h1 class="intro-title">
 
       <!--  Mission  -->
       <div class="section mission blue-section">
-        <div class="blue-section-inner">
-          <h2>Learn on the Go</h2>
-          <p>
-          As a journalist living in the digital age, a huge demand falls on you to be up to date with the latest in data gathering and processing skills.These courses are tailored to help you do just that.
+    <div class="blue-section-inner">
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra Indentation

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html [hunk 2]
```diff
@@ -25,15 +25,28 @@ <h1 class="intro-title">
...
           <br>
-          From learning Google apps that can make your investigative work easier to delving deeper into the intricacies of extracting and understanding data. You will become a head honcho in the latest techniques used in news gathering.
-          </p>
-        </div>
+          <br>
+          But our newsrooms are under-resourced, and don’t have the time and money to invest in high quality training to meet these new demands. In Africa, these problems compounded with the issues of low levels of internet access and smartphone adoption, and a chronic shortage of digital skills.
+          <br>
+          <br>
+          That’s why <a href="https://codeforafrica.org/">Code for Africa</a>, in partnership with <a href="http://newslab.withgoogle.com/">Google News Lab</a> and The <a href="http://worldbank.org/">World Bank</a>, has created the <a href="https://academy.codeforafrica.org/about">StoryLab Academy</a>.
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra for href target="blank"

Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra "The World Bank"

File: coursebuilder/modules/explorer/templates/explorer.html [hunk 3]
```diff
@@ -67,18 +67,17 @@
 
             <div class="col l4 m12 footer-social-icons">
                 <br>
-                <a href="https://plus.google.com/share?url=https://courses.academy.africa" target="_blank"
+                <a target="_blank"
```
Comment (chegejames) [inline coursebuilder/modules/explorer/templates/explorer.html]: @Sharonsyra Missing hrefs?

File: frontend/index.html [hunk 4]
```diff
@@ -97,12 +96,23 @@ <h1 class="intro-title">Free world-class digital journalism course with video ex
   <!--  Mission  -->
   <div class="section mission blue-section">
     <div class="blue-section-inner">
-      <h2>Learn on the Go</h2>
+      <h2>The challenge for African journalists
```
Comment (chegejames) [inline frontend/index.html]: @Sharonsyra indentation

File: frontend/scss/homepage/_main.scss [hunk 5]
```diff
@@ -10,26 +10,28 @@
...
+      max-width: 500px;
+      margin: 2.1rem 0 0 0;
     }
-
-    p {
-      color: $color-white;
-      font-weight: 400;
-      font-size: 18px;
-      margin: 40px 0;
+    a{
```
Comment (chegejames) [inline frontend/scss/homepage/_main.scss]: @Sharonsyra a {

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 1 above)
Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: Sorry on it

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 2 above)
Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: Thank you. I will use this

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: fixed thanks

File: coursebuilder/modules/explorer/templates/explorer.html (hunk 3 above)
Comment (Sharonsyra) [reply coursebuilder/modules/explorer/templates/explorer.html]: do we have these pages? Or i just have a dead link as a place holder?

File: frontend/scss/homepage/_main.scss (hunk 5 above)
Comment (Sharonsyra) [reply frontend/scss/homepage/_main.scss]: fixed thanks you.

File: frontend/scss/homepage/_main.scss [hunk 6]
```diff
@@ -57,10 +59,11 @@
 
 // Mission
 .mission {
+  background: white;
   .blue-section-inner {
```
Comment (chegejames) [inline frontend/scss/homepage/_main.scss]: @Sharonsyra blank line before.

File: frontend/scss/homepage/_main.scss [hunk 7]
```diff
@@ -57,10 +59,11 @@
...
     max-width: 1000px;
     text-align: center;
-
+    color: black;
```
Comment (chegejames) [inline frontend/scss/homepage/_main.scss]: @Sharonsyra blank line after

**Review by chegejames — CHANGES_REQUESTED** [review]
Your PR includes changes from a closed PR https://github.com/CodeForAfricaLabs/coursebuilder-core/pull/19

**Comment by Sharonsyra** [comment]
I added @andela-angene's content too in the PR but this has since been fixed

File: coursebuilder/modules/explorer/templates/explorer.html (hunk 3 above)
Comment (chegejames) [reply coursebuilder/modules/explorer/templates/explorer.html]: @Sharonsyra dead link until content is provided.

**Review by chegejames — APPROVED** [review]

File: coursebuilder/modules/explorer/_static/components/pages/templates/partners-template.html [hunk 8]
```diff
@@ -7,81 +7,44 @@
...
-              <p>
-                World Bank Group
-              </p>
-            </a>
-          </div>
-          <div class="item">
-            <a href="https://codeforafrica.org/">
-              <div class="partner-img">
+            <div class="item">
+              <a href="https://codeforafrica.org/">
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/pages/templates/partners-template.html]: @Sharonsyra target "blank"

File: frontend/index.html [hunk 9]
```diff
@@ -123,59 +123,38 @@ <h1 class="intro-title">
       <h2 class="section-label">Partners</h2>
       <div class="owl-carousel owl-theme awards-carousel">
           <div class="item">
-              <a href="https://www.facebook.com/HacksHackersAfrica">
-                <img src="images/hha.gif" alt=" Hack Hackers Africa">
-                <p>
-                  Hack Hackers Africa
-                </p>
-              </a>
+            <a href="https://codeforafrica.org/">
```
Comment (chegejames) [inline frontend/index.html]: @Sharonsyra target="_blank"

File: frontend/scss/homepage/_main.scss [hunk 10]
```diff
@@ -18,15 +18,15 @@
...
       max-width: 500px;
       margin: 0 0 0 0;
+      min-height: 430px;
     }
 
     a {
       background: red;
       position: absolute;
-      float: right;
       position: absolute;
```
Comment (chegejames) [inline frontend/scss/homepage/_main.scss]: @Sharonsyra get rid of one.

File: frontend/scss/homepage/_main.scss [hunk 11]
```diff
@@ -10,26 +10,30 @@
...
-      max-width: 700px;
-    }
     h1 {
-      font-size: 3rem;
+      font-size: 2rem;
+      font-family: Trebuchet MS;
       color: white;
+      background: rgba(0, 0, 0, 0.5);
+      padding: 80px;
+      max-width: 500px;
```
Comment (chegejames) [inline frontend/scss/homepage/_main.scss:23]: @Sharonsyra not looking good at on smaller screens i.e @media screen and (max-width: 992px)
.hero-banner .hero-banner-content h1

File: frontend/scss/homepage/_main.scss [hunk 12]
```diff
@@ -10,26 +10,30 @@
...
 
-    p {
-      color: $color-white;
-      font-weight: 400;
-      font-size: 18px;
-      margin: 40px 0;
+    a {
+      background: red;
+      position: absolute;
+      position: absolute;
```
Comment (chegejames) [inline frontend/scss/homepage/_main.scss]: @Sharonsyra get rid of one.

**Comment by chegejames** [comment]
@Sharonsyra please update the description and screenshots of the PR to include new features and changes.

**Pull request merged**